src/components/Table.jsx

```diff
@@ -0,0 +1,606 @@
+import { Column } from 'primereact/column';
+import { InputText } from 'primereact/inputtext';
+import { Dropdown } from 'primereact/dropdown';
+import { MultiSelect } from 'primereact/multiselect';
+import { Calendar } from 'primereact/calendar';
+import {workFlowsUrl} from '../constants/index';
+import Pagination from './Pagination';
+import AssigneEdit from '../logos/AssigneEdit.svg';
+import Edit from '../logos/Edit.svg';
+import ArrowSort from '../logos/ArrowSort.svg';
+import MoreIcons from '../logos/MoreIcons.svg';
+
+
+
+export default function Table({currentTab, setCustomers, customers, isAdmin, preText, nextText, currentPage, setCurrentPage}) {
+    const [searchByStyle, setSearchByStyle] = useState('');
+    const [searchByTitle, setSearchByTitle] = useState('');
+    const [searchByStatus, setSearchByStatus] = useState([]);
+    const [searchByAssignee, setSearchByAssignee] = useState('');
+    const [searchByUpdatedBy, setsearchByUpdatedBy] = useState("");
+    const [searchByUpdatedAt, setsearchByUpdatedAt] = useState(null);
+    const [selectedBrand, setSelectedBrand] = useState([]);
+    const [showFilters, setShowFilters] = useState(false);
+    const [selectedProducts, setSelectedProducts] = useState([]);
+    const [brands, setBrands] = useState([]);
+    const [assigneeList, setAssignee] = useState([]);
+    const [statuses, setStatus] = useState([]);
+    const [showEdit, setShowEdit] = useState(null);
+    const [styleSort, setStyleSort] = useState('desc');
+    const [titleSort, setTitleSort] = useState('desc');
+    const [brandSort, setBrandSort] = useState('desc');
+    const [statusSort, setStatusSort] = useState('desc');
+    const [updatedBySort, setUpdatedBySort] = useState('desc');
+    const [updatedAtSort, setUpdatedAtSort] = useState('desc');
+    const [assigneeSort, setAssigneeSort] = useState('desc');
+    const [currentSort, setCurrentSort] = useState('');
+    const [isRowSelected, setIsRowSelected] = useState(false);
+    const [isStatusSelected, setIsStatusSelected] = useState(false);
+
+    useEffect(() => {
+       fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=brand`, {
+                method: "POST",
+                headers: {
+                    "Content-type": "application/json; charset=UTF-8"
+                }
+            })
+            .then(response => response.json())
+            .then(result => setBrands(result?.data?.uniqueValues.filter(item=> item !== null).map(item=>({brand:item}))) );
+    }, []);
+    useEffect(() => {
+        fetch(`${workFlowsUrl}/search?limit=10&page=${currentPage}&unique=assignee`, {
+                 method: "POST",
+                 headers: {
+                     "Content-type": "application/json; charset=UTF-8"
+                 }
+             })
+             .then(response => response.json())
+             .then(result => setAssignee(result?.data?.uniqueValues.filter(item=> item !== null)));
```

Comment:
`result?.data?.uniqueValues.filter(Boolean)`
Change this in the above useEffect also.

Reply:
addressed